recipes/mongo-cxx-driver/all/conanfile.py

```diff
@@ -0,0 +1,112 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class MongoCxxConan(ConanFile):
+    name = "mongo-cxx-driver"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://mongocxx.org"
+    description = "C++ Driver for MongoDB"
+    topics = ("conan", "libbsoncxx", "libmongocxx", "mongo", "mongodb", "database", "db")
+    settings = "os", "compiler", "arch", "build_type"
+    exports_sources = ["CMakeLists.txt", "patches/**", "Find*.cmake"]
```

Comment:
```suggestion
    exports_sources = ["CMakeLists.txt", "patches/**", "Find*.cmake"]
```

There should be an way to patch those 2 lines here: https://github.com/mongodb/mongo-cxx-driver/blob/5b4628114ebddc9bd0092ea0eb3d66beef7818e9/src/bsoncxx/CMakeLists.txt#L82
Indeed, `mongo-c-driver`'s recipe doesn't model bson config file (because currently conan can only model one config file per recipe), so `cmake_find_package` (or better `cmake_find_package_multi`) puts bson target is in mongoc-1.0 config file.

Reply:
I think I made it